### PR TITLE
hide dino type in edit screen

### PR DESCRIPTION
**Pull request by LeoMcA**
https://jira.mozilla.com/browse/IAM-335?focusedCommentId=31976&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-31976

### CODE REVIEW

File: src/components/profile/edit/EditPersonalInfo.vue [hunk 1]
```diff
@@ -34,6 +34,7 @@
           :avatar="avatar"
           :size="264"
           :isStaff="scope.isStaff"
+          :edit="true"
```
Comment (andrew-sunada) [inline src/components/profile/edit/EditPersonalInfo.vue]: Why aren't we just removing this icon entirely from the editpersonalinfo.vue file? If we are hiding the picture always, why do we need to add a boolean flag to the userpicture?
Also, it doesn't seem appropriate that the UserPicture knows anything about any editing mechanism of the profile at the moment.

Comment (fiji-flo) [reply src/components/profile/edit/EditPersonalInfo.vue]: We could also rename the `isStaff` attribute into `showLabel` and just use that for everything. Thoughts?

**Comment by LeoMcA** [comment]
Do we want the "m" to appear on your usericon in the top right as well? There was a legacy `dinoType="Staff"` line in `TopBar.vue`.

**Comment by fiji-flo** [comment]
> Do we want the "m" to appear on your usericon in the top right as well? There was a legacy `dinoType="Staff"` line in `TopBar.vue`.

Yes I think it's nice to display it there. It helps me to figure out if I'm logged in as staff or volunteer.

**Pull request merged**